test/functions.jl

```diff
@@ -0,0 +1,8 @@
+using NNlib: glu
+
+@testset "glu" begin
+    x = [1 2 3; 4 5 6; 7 8 9; 10 11 12]	 
+    @test ceil.(glu(x,1)) == [1 2 3; 4 5 6]
+    @test_throws AssertionError glu(x,2)
+end
+
```

Comment:
Gradients should be tested as well, see https://github.com/FluxML/NNlib.jl/blob/542e5cdb537b727ca0f3b97e3bef7bf9cf897a3c/test/activations.jl#L224.